docs/how-to/distributed-training.txt

```diff
@@ -29,7 +29,7 @@ agent machine or across multiple machines.
 .. note::
 
       When the ``slots_per_trial`` option is changed, the per-slot batch size is set to
-      ``global_batch_size`` // ``slots_per_trial``.  The per-slot (per-GPU) and global batch size
+      ``global_batch_size // slots_per_trial``.  The per-slot (per-GPU) and global batch size
```

Comment:
question: what does // mean in this context?

Reply:
Division with truncation: https://www.python.org/dev/peps/pep-0238/#id22.